docs/source/models/segment_anything.rst

```diff
@@ -121,8 +124,11 @@ a given SAM model.
     # Load image
     image = load_image('./example.jpg', ImageLoadType.RGB8, device).float()
 
-    # Load the predictor
-    predictor = SamPredictor(sam_model)
+    # Load the prompter
+    prompter = SamPrompter(sam_model)
```

Comment:
If it’s SamPropter doesn’t make sense that you need to pass the model, it should load automatically. That must be part of the model factory

Reply:
so, should be a call of the `from pretrained` on the init and the user pass the args for it?